### DISCUSSION TITLE
proposal

### DISCUSSION THREAD

**Discussion by Clorasun** [DUpdateSystem/UpgradeAll#270] [General]
![S20920-12000807.png](https://user-images.githubusercontent.com/92291796/191164692-23384fe3-8ebb-4c35-9327-f43ff14c2554.png)
All here can be directly changed to show read application list, the application market calls the server so difficult to use, and hanging VPN will also check the failure, but the download and need VPN, contradictory, really do not understand how the application is to do this step, the application is quite marketable, but the development direction really does not work, the same check to update the application software APP Share this application can do, the application has considered pushing back to start over The application considered to push back?

**Comment by Pantyhose-X** [comment]
It does not get the list of installed applications.
It does not have the application package name , which is detected in the software source.

UpgradeAll I have 600 apps installed on my Android. Not a single update is available.
UpgradeAll is just a toy!

I now use [ApkTrack](https://github.com/JusticeRage/ApkTrack)
ApkTrack it doesn't have [apkpure](https://apkpure.com/) software source, kind of sucks

**Reply by xz-dev** [reply]
You need enable in app hub.
Also, you need use [CI version](https://t.me/upallci).

**Reply by xz-dev** [reply]
A other thing is Google Play is broken because Aurora API is down

**Reply by xz-dev** [reply]
And I'll try to find a way to easy adapte different website include apkpure.

**Reply by Pantyhose-X** [reply]
> And I'll try to find a way to easy adapte different website include apkpure.

You can refer to https://github.com/EFForg/apkeep , no need to re-write a lot of code

**Reply by Pantyhose-X** [reply]
> You need enable in app hub. Also, you need use [CI version](https://t.me/upallci).

Please create a group in the https://rvlt.gg

**Reply by xz-dev** [reply]
> > You need enable in app hub. Also, you need use [CI version](https://t.me/upallci).
> 
> Please create a group in the https://rvlt.gg

ok, I will, but I do not know how to admin it, do you want to be an administrator?
